tests/Test/Xrefcheck/AnchorsInHeadersSpec.hs

```diff
@@ -0,0 +1,31 @@
+{- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+module Test.Xrefcheck.AnchorsInHeadersSpec where
+
+import qualified Data.ByteString.Lazy as BSL
+import Test.Hspec (Spec, describe, it, shouldBe)
+
+import Xrefcheck.Core
+import Xrefcheck.Scanners.Markdown
+
+spec :: Spec
+spec = do
+    describe "\"ignore link\" mode" $ do
+        it "Check \"ignore link\" performance" $ do
```

Comment:
These test names seem to be wrong

Reply:
➕ 

It seems these 2 strings were forgotten to change